flask_app/models.py

```diff
@@ -120,7 +120,7 @@ class Session(db.Model, TypenameMixin, StatusPredicatesMixin, HasRelatedMixin, H
     num_error_tests = db.Column(db.Integer, default=0)
     num_skipped_tests = db.Column(db.Integer, default=0)
     num_finished_tests = db.Column(db.Integer, default=0)
-
+    num_interrupted_tests = db.Column(db.Integer, default=0)
```

Comment:
Please use `server_default="0"`, making it more efficient and more robust for migration. Regenerate the migration afterwards to make it reflect there